homeassistant/components/zha/button.py

```diff
@@ -103,3 +106,51 @@ def get_args(self) -> list[Any]:
         """Return the arguments to use in the command."""
 
         return [DEFAULT_DURATION]
+
+
+class ZHAButtonAttribute(ZhaEntity, ButtonEntity):
```

Comment:
Can we change this to `ZHAAttributeButton`

Reply:
Hehe, it was like that at first, but I didn't liked the double capital A. No problem it will be changed